### PR TITLE
DP-17676 tableau caption margins

### PR DESCRIPTION
**Pull request by ygannett**
**Description:**
Explain the technical implementation of the work done.

**Jira:**
https://jira.mass.gov/browse/DP-17750
(https://jira.mass.gov/browse/DP-17676)

**To Test:**
- [ ] Create/Edit an information details page.
- [ ] Add Tableau content.
Create_Information_Details___Mass_gov
- [ ] Find a rich text field for the caption.
caption_field
- [ ] Add content to the field and find you cannot add more than 500 characters.
- [ ] Save your edit and check the page.  Find the caption is rendered.
caption

**Screenshots/GIFs:**
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.

---

[Peer Review Checklist](https://github.com/massgov/opemass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -4,28 +4,30 @@
...
         "platform": {
-          "php": "7.1.18"
+            "php": "7.1.18"
         },
         "discard-changes": true
     },
-    "repositories": [
-        {
+    "repositories": {
+        "0": {
```
Comment (arthurbaghdas) [inline composer.json]: why these indexes where added?

File: composer.json [hunk 2]
```diff
@@ -205,7 +211,7 @@
         "drush/drush": "^10",
         "enyo/dropzone": "4.3.0",
         "guzzlehttp/guzzle": "^6.3",
-        "massgov/mayflower-artifacts": "9.49.1",
+        "massgov/mayflower-artifacts": "dev-patternlab/DP-17483_dataviz-textwrapping-container-suggestions",
```
Comment (arthurbaghdas) [inline composer.json]: Adding a comment here to keep track on this change

Comment (ygannett) [reply composer.json]: The MF version is changed back to 9.49.1.

File: composer.json (hunk 1 above)
Comment (ygannett) [reply composer.json]: reverted.

**Pull request merged**